packages/xo-server/src/api/vm.mjs

```diff
@@ -666,13 +666,26 @@ set.resolve = {
 
 // -------------------------------------------------------------------
 
-export async function restart({ vm, force = false }) {
+export const restart = defer(async function ($defer, { vm, force = false, forceBlockedOperation = false }) {
```

Comment:
Yes, I believe *bypass* is clearer.

Also, it should defaults to `force`.

The idea is for `bypass*` to avoid some specific checks, and for `force` to do anything necessary for the operation to succeed.

Example: https://github.com/vatesfr/xen-orchestra/blob/433851d7719b5aed1ad77b92dac5cd0514440fb9/%40xen-orchestra/xapi/vm.js#L342